core/templates/pages/exploration-editor-page/editor-tab/training-panel/training-data.service.spec.ts

```diff
@@ -166,15 +164,11 @@ describe('TrainingDataService', function() {
   });
 
   beforeEach(angular.mock.inject(function($injector, $rootScope) {
-    scope = $rootScope.$new();
-    $httpBackend = $injector.get('$httpBackend');
     siis = $injector.get('StateInteractionIdService');
     ecs = $injector.get('StateEditorService');
-    cls = $injector.get('ChangeListService');
     ess = $injector.get('ExplorationStatesService');
     rs = $injector.get('ResponsesService');
     tds = $injector.get('TrainingDataService');
-    IS = $injector.get('INTERACTION_SPECS');
```

Comment:
Can we remove these lines too?

https://github.com/oppia/oppia/blob/b1ba8d11e81b968cddaaa66180f8b787e7c72372/core/templates/pages/exploration-editor-page/editor-tab/training-panel/training-data.service.spec.ts#L102-L107

Reply:
Done.